schema-layer/data-structures/Data.md

```diff
@@ -0,0 +1,62 @@
+`Data` is an advanced layout for representing binary data.
+
+It is flexibible enough to support very small and very large (multi-block) binary data.
```

Comment:
flexible :)

Reply:
It’s so flexible, it’s biblical: flexibible.